test/sub_merger_test.rb

```diff
@@ -0,0 +1,165 @@
+require 'test_helper'
+
+class SubMergerTest < Minitest::Test
+
+  DUMMY_KEEP_PARSE = {
```

Comment:
this needs to be:

```ruby
  DUMMY_KEEP_PARSE = {
    "type" => "Physical",
    'name' => 'physical_example',
    'facts' => {
      "distribution::name" => "Red Hat Server",
      "distribution::version" => "6.1"
    }
  }
```

changes:
* `type` inside the `host`
* `host` itself dropped